src/raftstore/store/fsm/peer.rs

```diff
@@ -1689,6 +1709,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             }
         }
 
+        LAST_GC_VEC_LEN.with(|last_gc_vec_len| *last_gc_vec_len.borrow_mut() = gc_vec.len());
```

Comment:
Can there be some comments explaining why we need to do this? 🤔

Reply:
I add some just now.